docs/plugins/harbor.md

```diff
@@ -0,0 +1,93 @@
+# harbor Plugin
+
+This plugin installs [harbor](https://goharbor.io/) in an existing Kubernetes cluster using the Helm chart.
+
+## Usage
+
+##### External storage
```

Comment:
"Usage" is a level 2 subtitle, so "External storage" should be a level 3 subtitle. Same for other subtitles.

Also, for titles and subtitles, please use [title case](https://en.wikipedia.org/wiki/Title_case). Try [this tool](https://titlecaseconverter.com/) maybe.